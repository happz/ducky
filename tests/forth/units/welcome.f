: TEST
  DECIMAL
  ." DuckyFORTH VERSION " VERSION . CR
  UNUSED . ." CELLS REMAINING" CR
  ." OK " CR
;

TEST
