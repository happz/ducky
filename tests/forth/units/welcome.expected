DuckyFORTH VERSION 2 
1537 CELLS REMAINING
OK 
